### DISCUSSION TITLE
Create a pull-request template

### DISCUSSION THREAD

**Discussion by zepfred** [TimefoldAI/timefold-solver#502] [Ideas]
The goal is to create a standardized process for updating quickstart modules and a pull request (PR) template that includes checkbox fields that must be marked before submitting or merging a PR. Here are some examples:

1. Does the proposed change, if needed, incorporate tests to prevent any regressions?
2. Are the UI and JS files fully tested?
    - Did you check all UI buttons (E.g., solve and analyze buttons)? 
    - Did you check the network calls?
    - Did you check the console messages?
3. Do you have a green build?

**Comment by triceo** [comment]
Moved here:
https://github.com/TimefoldAI/timefold-quickstarts/issues/200
